lib/mix/tasks/compile.make.ex

```diff
@@ -9,6 +9,20 @@ defmodule Mix.Tasks.Compile.ElixirMake do
 
   ## Configuration
 
+  The configuration options are listed below. These need to be added to
+  the `project` function in the `mix.exs` file, as in this example:
+
+      def project do
+        [
+          app: :myapp,
+          make_executable: "make",
+          make_makefile: "Othermakefile",
+          compilers: [:elixir_make] ++ Mix.compilers,
+        ]
+      end
```

Comment:
It would be better to use the style used in the `mix.exs` generated by `mix new`:

```elixir
[app: :myapp,
 ...,
 make_executable: "..."]
```

Wdyt?